admin/views/tool-bulk-editor.php

```diff
@@ -63,7 +63,7 @@ function wpseo_get_rendered_tab( $table, $id ) {
 
 ?>
 <script>
-	var wpseoBulkEditorNonce = '<?php echo wp_create_nonce( 'wpseo-bulk-editor' ); ?>';
+	var wpseoBulkEditorNonce = '<?php echo wp_json_encode( wp_create_nonce( 'wpseo-bulk-editor' ) ); ?>';
```

Comment:
As `wp_json_encode` wraps a string with quotes, this will output a double qouted string `'"c84ce9bd71"'`.
Could you remove the wrapping single quotes?

Reply:
Ouch.. very sneaky and looks like this is not documented behaviour.

Thanks for catching that. I've added a second commit to the PR to fix this. Feel free to squash them on merge.

I think the documentation snaffu for the `wp_json_encode()` method should be reported in WP Core trac.